http/go/nethttp/kiota_client_factory.go

```diff
@@ -0,0 +1,36 @@
+package nethttplibrary
+
+import (
+	nethttp "net/http"
```

Comment:
Why are we aliasing to a longer name?

Reply:
that's a practice I started in downstream packages to differentiate easily between kiota http and net http. I decided to align on that so it's easier to know what we're referring to in any give context.

Reply:
Aliasing to a longer name is non-idiomatic we could keep kiota http and make net http just http not a blocker but something we should consider doing.